src/main/java/com/jfrog/ide/common/utils/Utils.java

```diff
@@ -67,7 +68,8 @@ public static Issue toIssue(com.jfrog.xray.client.services.summary.Issue other)
             VulnerableComponents vulnerableComponents = vulnerableComponentsList.get(0);
             fixedVersions = vulnerableComponents.getFixedVersions();
         }
-        return new Issue(other.getDescription(), severity, other.getSummary(), fixedVersions);
+        String cve = ListUtils.emptyIfNull(other.getCves()).stream().map(Cve::getId).filter(StringUtils::isNotBlank).findAny().orElse("");
```

Comment:
Let's explain why we're picking up the first CVE and also explain why. Also, since this logic seems to be repeating twice in the code, I suggest moving it into a util.